models/base/src_base.yml

```diff
@@ -0,0 +1,267 @@
+version: 2
+
+sources:
+  - name: atomic
+    schema: "{{ var('snowplow__atomic_schema', 'atomic') }}"
```

Comment:
You'll need to adjust for the intergration tests schema, because of the way things in dbt work with compile vs runtime stuff, see ecommerce for an example https://github.com/snowplow/dbt-snowplow-ecommerce/blob/5cc661a5410a81ec83d102e8db873f19e6624a05/models/base/src_base.yml#L5